service/src/main/kotlin/fi/espoo/evaka/shared/async/AsyncJob.kt

```diff
@@ -204,6 +204,15 @@ sealed interface AsyncJob : AsyncJobPayload {
         override val user: AuthenticatedUser? = null
     }
 
+    data class SendCitizenBasicDocumentNotificationEmail(
+        val documentId: ChildDocumentId,
+        val childId: ChildId,
+        val recipientId: PersonId,
+        val language: Language,
+    ) : AsyncJob {
+        override val user: AuthenticatedUser? = null
+    }
```

Comment:
Puhuttu jo aiemmin f2f: Tässä olisi parempi käyttää olemassaolevaa async-jobia ja lisätä siihen vain enum-kenttä joka kertoo mikä teksti halutaan. Näin vältetään duplikaatio. Kentällä täytyy olla default-arvo jotta kannassa jo olemassaolevien async_job -rivien deserialisaatio toimii yhä.